packages/cli/README.md

```diff
@@ -1 +1,246 @@
-# cli
+# Bucket CLI
+
+Command-line interface for interacting with Bucket services. The CLI allows you to manage apps, features, authentication, and generate TypeScript types for your Bucket features. With this tool, you can streamline your feature flagging workflow directly from your terminal.
+
+## Installation
+
+You can install the Bucket CLI either locally to your project or globally on your system.
+
+### Local Installation (Recommended)
+
+Local installation is recommended for project-specific usage and ensures consistent CLI versions across your team.
+
+```bash
+# npm
+npm install --save-dev @bucketco/cli
+
+# yarn
+yarn add --dev @bucketco/cli
+
+# pnpm
+pnpm add --save-dev @bucketco/cli
+```
+
+When installed locally, use the CLI with npx:
+
+```bash
+# npm
+npx bucket <command>
```

Comment:
You don't even need to install it if you're using `npx` i believe?

Reply:
You would have to run it with the full package name without installing it. Would you like me to add that as a suggestion?
```bash
npx @bucketco/cli <command>
```

Reply:
Clarified this, need to test to see what is possible when published

Reply:
OK, lets test once the package exists.